addon/components/frost-popover.js

```diff
@@ -22,7 +22,6 @@ export default Component.extend(PropTypeMixin, {
     excludePadding: PropTypes.bool,
     handlerIn: PropTypes.string,
     handlerOut: PropTypes.string,
-    includeContentInEvents: PropTypes.bool, // making this true lets the content also inheret events properly
```

Comment:
What's the reason for removing this. Wouldn't it be considered a #MAJOR# if you removed a feature.

Reply:
So, normally yes. But this was added just for one feature in mcp recently, and was not documented. So, no one else is using it, and now mcp isn't either. And the functionality is brought forward - but you can't turn it off. If we feel this should be behind a flag, we can do that, but it prob shouldn't be called this. And I don't think it should be something you can turn off - when would you specifically not want to be able to hover/select the text in a popover?

Reply:
I think @EWhite613 may have added this? Even if this flag isn't used in MCP, it might be used in other apps. Let's be on the safe side and make this a #MAJOR#.

Reply:
ok